pkg/apis/workflow/v1alpha1/workflow_types.go

```diff
@@ -318,6 +328,9 @@ type WorkflowSpec struct {
 
 	// Synchronization holds synchronization lock configuration for this Workflow
 	Synchronization *Synchronization `json:"synchronization,omitempty" protobuf:"bytes,35,opt,name=synchronization,casttype=Synchronization"`
+
+	// VolumeGC describes the strategy to use when to deleting volumes from completed workflows
+	VolumeGC *VolumeGC `json:"volumeGC,omitempty" protobuf:"bytes,36,opt,name=volumeGC,casttype=VolumeGC"`
```

Comment:
as this only affects PVCs, I wonder if it should be called `PVCGC`?

Reply:
I'm not tied to the name `volumeGC` at all.  That said, this strategy is directly tied to the `volumeClaimTemplates` property, so I felt that this name might've been a little clearer about what it was related to.

Maybe `volumeClaimGC`?  Alternatively, it could be interesting to add `gcStrategy` as a field under the actual `volumeClaimTemplate`, although that would mean diverging from the k8s definition.

Reply:
I agree volmeGC is _technically_ not correct, so I think to be consistent with `volumeClaimTemplates`, I agree with `volumeClaimGC`

Reply:
renamed this to `volumeClaimGC`